test/Web3Spec/Encoding/DataSpec.purs

```diff
@@ -0,0 +1,59 @@
+module Web3Spec.Encoding.DataSpec (dataMakerSpec) where
+
+-- There is no test here, this just needs to compile
```

Comment:
Should we still have this comment? I'm asking as we have `shouldEqual` so it's a test.